lib/cosmos/dart/script/dart_stream_server.rb

```diff
@@ -0,0 +1,121 @@
+require File.expand_path('../../config/environment', __FILE__)
+require 'dart_common'
+require 'dart_logging'
+class DartStreamServer < Cosmos::TcpipServerInterface
```

Comment:
The DART decom server uses the JsonDRb class from COSMOS which is the same code which powers the CmdTlmServer. Why is this just using the TcpipServerInterface?

Reply:
Because it streams packets.   JsonDRb is just a single command/response.